build/STRINGER.bzl

```diff
@@ -12,7 +12,8 @@ def stringer(src, typ, name):
       cmd = """
          GO_REL_PATH=`dirname $(location @go_sdk//:bin/go)`
          GO_ABS_PATH=`cd $$GO_REL_PATH && pwd`
-         env PATH=$$GO_ABS_PATH HOME=$(GENDIR) \
+         # Set GOPATH to something to workaround https://github.com/golang/go/issues/43938
```

Comment:
Does this break the Windows build?

Reply:
This is a good  question... I haven't explicitly checked it. I hope it just works, because we use Unix slashes in the same script.

Reply:
The build passed, you're good :)